modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,585 @@
+
+* Conflicts must not be logged for the collection `agents.retailers`.
+
+* Conflict logs for the `inventory` scope must be logged to a collection called `conflictlogsbucket2.myappscope.inventorylogs`.
+
+* Conflict logs for the scopes and collections unaffected by `loggingRules` are stored in the default conflict log collection `conflictlogsbucket1.myappscope.log1collection`.
+
+[source,json]
+----
+--data-urlencode conflictLogging= '{
+"disabled": false,
+"bucket": "conflictlogsbucket1",
+"collection": "myappscope.log1collection"
+"loggingRules": {
+      "agents.partners": {
+        "bucket": "conflictlogsbucket2",
+        "collection": "myappscope.partnerlogs"
+      },
+      "agents.retailers": null,
+      "inventory": {
+        "bucket": "conflictlogsbucket2",
+        "collection": "myappscope.inventorylogs"
+      }
+    }
+}'
+----
+
+[#disabled-with-stored-logging-rules]
+==== Disabled with logging rules configured
+
+You can configure `conflictLogging` (specify the default conflict collection and the `loggingRules`) but not enable `conflictLogging` by setting `disabled` to `true`. 
+
+NOTE: You can update the replication and set the `conflictLogging` status `disabled` to `true` or `false` at any time.
+
+When you set `"disabled": true` and have specified other configurations, then the specified values are retained.
+
+*Format*:
+
+[source,json]
+----
+--data-urlencode conflictLogging= '{
+                    "bucket": [bucketName],
+                    "collection": [scope].[collection],
+                    "loggingRules": {[logging_rules]},
+                    "disabled": true,
+}'
+----
+
+*Example*: Conflict logging is disabled, but the default conflict collection and the logging rules are retained.
+
+[source,json]
+----
+--data-urlencode conflictLogging= '{
+                    "disabled": true,
+                    "bucket": "conflictlogsbucket1",
+                    "collection": "myappscope.log1collection",
+                    "loggingRules": {
+                        "agents.partners": {
+                        "bucket": "conflictlogsbucket2",
+                        "collection": "myappscope.partnerlogs"
+                    },
+                    "agents.retailers": null,
+                    "inventory": {
+                        "bucket": "conflictlogsbucket2",
+                        "collection": "myappscope.inventorylogs"
+                    }
+            }
+}'
+----
+
+[#logging-rules]
+=== loggingRules in conflictLogging
+
+Use `loggingRules` to override the xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#enabled-for-all-conflicts[default conflict logging settings].
+
+NOTE: Conflict logging rules can be saved without enabling the Conflict Logging setting, however, the logging rules come into effect when the Conflict Logging setting is enabled.
```

Comment:
Use conflictLogging setting instead of Coflict Logging setting to be consistent with other usage in this page.

NOTE: Conflict logging rules can be saved without enabling the conflictLogging setting, however, the logging rules come into effect when the conflictLogging setting is enabled.